src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public interface Figure {
```

Comment:
create a separate interface for each behaviour - Drawable, AreaCalculator. Let `color` just be a field in abstract parent class. Remove this file